harness/features/initialize.local.test.ts

```diff
@@ -84,6 +84,76 @@ describe('Initialize Tests - Local', () => {
                 await wait(3100)
                 await testClient.close()
             }, 5000)
+
+            it('uses the same config if the etag matches', async () => {
+                const testClient = new LocalTestClient(name)
+                scope
+                    .get(`/client/${testClient.clientId}/config/v1/server/${testClient.sdkKey}.json`)
+                    .reply(200, config)
+
+                scope
+                    .get(`/client/${testClient.clientId}/config/v1/server/${testClient.sdkKey}.json`)
+                    .matchHeader('If-None-Match', (value) => {
+                        return true
+                    })
+                    .reply(304, {})
+
+                await testClient.createClient(true, { configPollingIntervalMS: 1000 })
+
+                expect(scope.pendingMocks().length).toEqual(1)
+
+                await wait(1100)
+                expect(scope.pendingMocks().length).toEqual(0)
+                // make sure the original config is still in use
+                const variable = await testClient.callVariable(shouldBucketUser, 'number-var', 0)
+                expect((await variable.json()).data.value).toEqual(1)
+                await testClient.close()
+            })
+
+            it('uses the same config if the refetch fails, after retrying once', async () => {
+                const testClient = new LocalTestClient(name)
+                scope
+                    .get(`/client/${testClient.clientId}/config/v1/server/${testClient.sdkKey}.json`)
+                    .reply(200, config)
+
+                scope
+                    .get(`/client/${testClient.clientId}/config/v1/server/${testClient.sdkKey}.json`)
+                    .times(2)
+                    .reply(503, {})
+
+                await testClient.createClient(true, { configPollingIntervalMS: 1000 })
```

Comment:
maybe increase the polling interval so that we're sure the second time the mock endpoint is hit it's actually from the retry and not polling.